packages/usa-nav/src/styles/_usa-nav.scss

```
@@ -475,19 +475,22 @@ $expand-less-icon: map-merge(
 }
 
 // Safari-only fix that forces a vertical scrollbar when mobile menu is open.
-// Only needed in the 15px immediately preceding $theme-header-max-width.
+// Only needed in the 15px immediately preceding $theme-header-min-width.
 //
 // Note: 15px is the current width of the Safari scrollbar.
 // More details in https://github.com/uswds/uswds/pull/5443
-$safari-header-bug-min-width: calc(
-  units($theme-header-max-width) - px-to-rem(15px)
-);
 
-@media (min-width: $safari-header-bug-min-width) {
-  .usa-js-mobile-nav--active.is-safari {
-    overflow-y: scroll;
-    position: fixed;
-    // --scrolltop set with JS with zero as fallback.
-    top: var(--scrolltop, 0);
+@if $theme-header-min-width != "none" {
+  $safari-header-bug-min-width: calc(
+    units($theme-header-min-width) - px-to-rem(15px)
+  );
+
+  @media (min-width: $safari-header-bug-min-width) {
+    .usa-js-mobile-nav--active.is-safari {
+      overflow-y: scroll;
+      position: fixed;
+      // --scrolltop set with JS with zero as fallback.
+      top: var(--scrolltop, 0);
+    }
```

Quick note that I was also playing with this to make it a bit more resilient. Might be overkill for the need though since the Sass spits out warnings for non-named breakpoint values. This would just prevent negative values in the media query if someone were to set the value for `$theme-header-min-width` to something like 0. 
```suggestion
@if map-has-key($project-spacing-named, $theme-header-min-width) {
  $safari-header-bug-min-width: calc(
    units($theme-header-min-width) - px-to-rem(15px)
  );

  @media (min-width: $safari-header-bug-min-width) {
    .usa-js-mobile-nav--active.is-safari {
      overflow-y: scroll;
      position: fixed;
      // --scrolltop set with JS with zero as fallback.
      top: var(--scrolltop, 0);
    }
  }
}

```

---

Another update:  see [this comment](https://github.com/uswds/uswds/pull/5624#discussion_r1419315542)